css/style.css

```diff
@@ -2348,7 +2363,8 @@ input#featured[type="checkbox"] {
 
 .content-expander.visible {
```

Comment:
Please pull latest main branch and exclude fixes around content expander as these should be already there. Thanks!